client/containers/Pub/PubSubmissionHeader/PubSubmissionHeader.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import PubSubmissionHeader from './PubSubmissionHeader';
+// import { locationData, loginData, communityData } from 'utils/storybook/data';
+
+storiesOf('containers/Pub/PubSubmissionHeader', module).add('default', () => (
+	<PubSubmissionHeader />
```

Comment:
This component will need to take a `Submission` as a prop, so I'd suggest that you start constructing a fake one here:

```ts
const submission: Submission = {
    instructionsText: { ... },
    // and all the rest
}
```

Since this type has so many `DocJson` properties you might find the `getEmptyDoc()` function in `components/Editor` helpful for spoofing them. And if you really, you can probably get away with defining the `submission` type as `Pick<Submission, 'a' | 'b' | 'c'>` where `'a' | 'b' | 'c'` are only the properties you currently need (which for now is just the `instructionsText`.